docs/sources/administration/plugin-management/_index.md

```diff
@@ -164,6 +175,8 @@ plugins:
 
 When the update is complete, a confirmation message will indicate the installation was successful.
 
+## Manage your plugins
+
 ### Update a plugin
```

Comment:
opportunity to say that we strongly recommend running latest and optionally link to advisor or other ways to tell if you are not running latest